assets/js/everypay.js

```diff
@@ -27,41 +46,53 @@ let payformResponseHandler = (response) => {
     }
 };
 
-function load_everypay() {
+function checkIfTokenizedCardHasTheRequiredFields(tokenizedCard) {
+    if (!tokenizedCard.hasAttribute('crd') ||
+        !tokenizedCard.hasAttribute('card_type') ||
+        !tokenizedCard.hasAttribute('exp_month') ||
+        !tokenizedCard.hasAttribute('exp_year') ||
+        !tokenizedCard.hasAttribute('last_four')
+    ) {
+        return false;
+    }
+    return true;
+}
 
+function load_everypay() {
     modal.show_loading();
-    let payload = create_payload(EVDATA);
-
+    var payload = create_payload(EVDATA);
+    // @note
+    console.log('check', {payload})
+    if (!payload) {
+        alert("An error occurred. Please try again.");
```

Comment:
Is alert a valid error handling or just forgotten?

Reply:
it was there previously, in not mine. I believe is a valid alert